docs/version0.44.md

```diff
@@ -31,6 +31,7 @@ The following new features and notable changes since version 0.43.0 are included
 - [XL C++ Runtime 16.1.0.7 or later required for AIX OpenJ9 builds on OpenJDK 8](#xl-c-runtime-16107-or-later-required-for-aix-openj9-builds-on-openjdk-8)
 - [New `-XX:[+|-]ShowUnmountedThreadStacks` option added](#new-xx-showunmountedthreadstacks-option-added)
 - [VMID query in the `jcmd` tool enhanced](#vmid-query-in-the-jcmd-tool-enhanced)
+- [Reference to the field names in `J9BuildFlags` changed in the DDR code](#reference-to-the-field-names-in-j9buildflags-changed-in-the-ddr-code)
```

Comment:
Besides the grammar problem (it would say "references", not "reference"), this isn't a fair summary; suggest: "DDR field names in J9BuildFlags changed".